src/main/java/refactoringml/RunImportCsv.java

```diff
@@ -0,0 +1,165 @@
+package refactoringml;
```

Comment:
Can you please transfer this into a new PR? It is unrelated to the other proposed changes? And I find it difficult to assess.